javascript/ql/lib/semmle/javascript/security/IncompleteMultiCharacterSanitization.qll

```diff
@@ -0,0 +1,202 @@
+/**
```

Comment:
Should this file instead be named `IncompleteMultiCharacterSanitizationQuery.qll`? Such that we use the `Query.qll` suffix. 

Otherwise LGTM.